### PR TITLE
Rework tests to make negate required in QueryRepresentation

### PR DESCRIPTION
**Pull request by Ladsgroup**
Refactoring the tests so it would produce a simple condition from
property and value and then overriding the defaults if needed.

Also, cleaning up other test to use a function as well.

### CODE REVIEW

File: tests/unit/sparql/buildQuery.spec.ts [hunk 1]
```diff
@@ -1,63 +1,62 @@
...
+		const simpleCondition: Condition = {
+			propertyId,
+			value,
+			datatype: 'string',
+			propertyValueRelation,
+			subclasses: false,
+			negate: false,
+		};
+
+		return JSON.parse( JSON.stringify( simpleCondition ) );
```
Comment (micgro42) [inline tests/unit/sparql/buildQuery.spec.ts]: This JSON.parse and JSON.stringify should not be necessary. I think this method always returns a fresh now object, not a reference to the same.

File: tests/unit/store/getters.spec.ts [hunk 2]
```diff
@@ -3,44 +3,46 @@ import getters from '@/store/getters';
...
+				conditionId: '0.123',
+				subclasses: false,
+				negate: false,
+			} ],
+			limit: 0,
+			useLimit: false,
+			omitLabels: true,
+			errors: [],
+		};
+		return JSON.parse( JSON.stringify( simpleRootState ) );
```
Comment (micgro42) [inline tests/unit/store/getters.spec.ts]: Also that JSON.parse and JSON.stringify shouldn't be needed. Just return the object :)

**Review by micgro42 — CHANGES_REQUESTED** [review]
We can make it a bit simpler :)

**Comment by Ladsgroup** [comment]
> We can make it a bit simpler :)

Done. Thanks!

**Review by micgro42 — APPROVED** [review]

**Pull request merged**